src/components/sandbox/contract-call/index.tsx

```diff
@@ -129,7 +129,7 @@ export const ContractCall: React.FC<any> = ({ showTransactionDialog, ...rest })
           in={showSearch}
         >
           {styles => (
-            <Box width="100%" position="absolute" style={styles}>
+            <Box width="100%" position="absolute" style={styles as any}>
```

Comment:
Lot of lines like this. What the deal with styles here, it's more strictly typed? Can styles be typed to something in the `csstype` lib? 🤔